### PR TITLE
Convert scala templates to groovy templates

### PR DESCRIPTION
**Pull request by manuelbernhardt**

### CODE REVIEW

**Comment by geralddejong** [comment]
Testing:

No big problems encountered.  Looks like the templating is fine, and now we can move towards refining the interface at a faster pace.
- logged in
- created new object
- object view after save has a big text area and two save buttons
- create a collection shows a strange "Test" button at the top
- created story, seemed a little precarious to have to save story after creating pages separately
- edit page of story doesn't work first time.  when viewing story again, it does work. (tried this twice, it's true)
- story thumbnail set by what appears to be editing a page, seems strange

**Comment by kiivihal** [comment]
I will review it now as well

**Comment by kiivihal** [comment]
I get the following problem with file upload.

Internal Server Error (500) for request POST /user.fileupload/uploadfile?uid=a09fd225-bd71-49c0-aac4-248aa25ca016

Execution exception (In /app/controllers/user/FileUpload.scala around line 33)
NullPointerException occured : null

play.exceptions.JavaExecutionException
    at play.mvc.ActionInvoker.invoke(ActionInvoker.java:231)
    at Invocation.HTTP Request(Play!)
Caused by: java.lang.NullPointerException
    at controllers.user.FileUpload$.uploadFile(FileUpload.scala:33)
    at play.mvc.ActionInvoker.invokeWithContinuation(ActionInvoker.java:548)
    at play.mvc.ActionInvoker.invoke(ActionInvoker.java:502)
    at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:496)
    at play.mvc.ActionInvoker.invokeControllerMethod(ActionInvoker.java:473)
    at play.mvc.ActionInvoker.invoke(ActionInvoker.java:161)

**Comment by kiivihal** [comment]
I tested everything else in chrome and everything is a lot more snappy. Except the upload problem I reported above everything else seems to be functioning exactly the way it was before. I was able to create and edit everything without a hitch. There are some interface/css issue where things just don't look right. But I am sure this is being taken care off in Eric's branch.

So as far as I am concerned this can be merged into the master branch.

File: app/controllers/user/FileUpload.scala [hunk 1]
```diff
@@ -26,10 +26,10 @@ object FileUpload extends DelvingController with Secure {
...
     val uploads: List[Upload] = request.args.get("__UPLOADS").asInstanceOf[java.util.List[play.data.Upload]]
-    uploadFile(uid, asScalaIterable(uploads))
+    uploadFileInternal(uid, asScalaIterable(uploads))
   }
 
-  def uploadFile(uid: String, uploads: Iterable[Upload]): Result = {
+  @Util def uploadFileInternal(uid: String, uploads: Iterable[Upload]): Result = {
     val uploadedFiles = for (upload: play.data.Upload <- uploads) yield {
       val f = fs.createFile(upload.asStream())
       f.filename = upload.getFileName
```
Comment (manuelbernhardt) [inline app/controllers/user/FileUpload.scala:35]: Sjoerd, here is the fix for the issue you reported - maybe you have a local change so this fix was not merged in?

**Pull request merged**